docs/general/serializers.md

```diff
@@ -74,6 +74,30 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+Polymorphic relationships are serialized just by specifying the relationship. For example:
```

Comment:
My question is: should it, though? I lack experience with polymorphic associations, but is it expected that the client needs not know the type of associated resources in that case?